api/specs/web-server/_catalog.py

```diff
@@ -130,3 +131,15 @@ def get_service_resources(
     _params: Annotated[ServicePathParams, Depends()],
 ):
     ...
+
+
+@router.get(
+    "/catalog/services/{service_key:path}/{service_version}/pricing-plan",
+    response_model=Envelope[ServicePricingPlanGet],
+    summary="Retrieve default pricing plan for provided service",
+    tags=["pricing-plans"],
```

Comment:
NOTE: by adding extra tag, it will appear twice in the API. Which is ok, but just FYI